jsonloader/parser2v2/parse_other_license_test.go

```diff
@@ -45,9 +45,20 @@ func TestJSONSpdxDocument_parseJsonOtherLicenses2_2(t *testing.T) {
 		},
 	}
 
+	otherLicensestest2 := []byte(`{
+		"hasExtractedLicensingInfos":[{
+			"extractedText" : "\"THE BEER-WARE LICENSE\" (Revision 42):\nphk@FreeBSD.ORG wrote this file. As long as you retain this notice you\ncan do whatever you want with this stuff. If we meet some day, and you think this stuff is worth it, you can buy me a beer in return Poul-Henning Kamp  </\nLicenseName: Beer-Ware License (Version 42)\nLicenseCrossReference:  http://people.freebsd.org/~phk/\nLicenseComment: \nThe beerware license has a couple of other standard variants.",
+			"licensd" : "LicenseRef-Beerware-4.2"
```

Comment:
It looks like this might be a typo -- `licensd` => `license` -- or was this typo intentional for the test?

Reply:
Discussed on call, this was intentional -- all good, thanks!